src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -301,19 +316,23 @@ export class RetrospectiveFeedbackComponent implements OnInit, OnChanges, OnDest
         }
     }
 
-    getTeamMemberOptions(teamMembers) {
+    getTeamMemberOptions() {
+        let teamMembers = this.teamMembers;
         teamMembers = _.map(teamMembers, (data: any) => {
             return {
                 id: _.parseInt(data.ID),
                 value: (data.FirstName + ' ' + data.LastName).trim(),
             };
         });
-        return [{ id: null, value: 'None' }, ...teamMembers];
+        return teamMembers;
     }
 
-    revertCellValue(params) {
+    revertCellValue(params, assigneeIDPrevValue) {
         const rowData = params.data;
         rowData[params.colDef.field] = params.oldValue;
+        if (params.colDef.field === 'Scope' && params.newValue === RETRO_FEEDBACK_SCOPE_TYPES.Team) {
```

Comment:
Do we also have a `oldValue` field, just like `newValue`? If yes, shouldn't we use that?

Reply:
done